Bottom: 8541dcb4fb80fbbadd920786eb285165ff989c54
Top:    b64d7bdafd93e3bda65d32822384fcc1ae63a813
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 14:13:31 +0000

Disable polygon heritage


---

diff --git a/src/polygon1.c b/src/polygon1.c
index b9c728c..3dc4083 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2825,8 +2825,10 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
   *res = NULL;
 
   /* Make copies for tracking polygon parentage (DEBUG) */
-  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
-      return err_no_memory;
+//  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
+//      return err_no_memory;
+  a_copy = NULL;
+  b_copy = NULL;
 
   /* Move the parentage information over onto the copy */
   if (a_copy != NULL)
